crates/protect-ffi/src/lib.rs

```diff
@@ -351,6 +366,8 @@ async fn decrypt(
         .zerokms
         .decrypt_single(
             encrypted_record,
+            // Specifying None here will result in the client using the keyset identifier from the client
+            None,
```

Comment:
Is the  intent of this change to support multitenancy in Protect.js? In that case would we want to allow for setting the keyset ID in encrypt/decrypt calls, too?

Reply:
From memory, if a keyset ID was provided at encryption time, then the keyset ID will be encoded into the ciphertext. `cipherstash-client` will handle that automatically.

Reply:
I'd like the Decrypt options to be the same IdentifiedBy struct as the client, but right now it's expecting a Uuid. Once we fix that in cipherstash-client I'll make a follow up PR.